packages/shared/styles/components/SfImage.scss

```
@@ -25,4 +25,26 @@ $image__overlay-color: $c-white !default;
     background-color: $image__overlay-background-color;
     color: $image__overlay-color;
   }
+  &__picture{
+    display: block;
+    opacity: 0;
+    &:first-child{
+      opacity: 1 !important;
```

acho que não precisa deste !important

```suggestion
      opacity: 1;
```